LICENSE

```diff
@@ -1,6 +1,6 @@
 MIT License
```

Comment:
In your app's root directory, can you also add in a `.github` folder and files in it like the Bolt Python starter template has here? https://github.com/slack-samples/bolt-python-starter-template/tree/main/.github

This will make sure the app has dependabot as well as any workflows related to the python template added in for your app as well. Another thing to note is that when you copy this over you should update the `CODEOWNERS` file in that directory to add in your GitHub username so you will be tagged in any future PRs or updates made to this code!